breaking/entrypoint.sh

```diff
@@ -23,7 +23,32 @@ fi
 echo "flags: $flags"
 
 if [ -n "$flags" ]; then
-    oasdiff breaking "$base" "$revision" $flags
+    output=$(unbuffer oasdiff breaking "$base" "$revision" $flags)
 else
-    oasdiff breaking "$base" "$revision"
+    output=$(unbuffer oasdiff breaking "$base" "$revision")
 fi
+
+echo "$output"
+
+# GitHub Actions limits output to 1MB
+# We count bytes because unicode has multibyte characters
+size=$(echo "$output" | wc -c)
+if [ "$size" -ge "1000000" ]; then
+    echo "WARN: Diff exceeds the 1MB limit, truncating output..." >&2
+    output=$(echo "$output" | head -c $1000000)
+fi
+
+delimiter=$(cat /proc/sys/kernel/random/uuid | tr -d '-')
```

Comment:
Is it the container delimiter? If yes, is it always the same?

Reply:
It's unique every time the script is run, to avoid terminating the heredoc below early. Scripts typically use `EOF` as a marker, but such a short token could easily occur in the oasdiff output.